.editorconfig

```diff
@@ -17,8 +17,6 @@ ij_kotlin_name_count_to_use_star_import = 2147483647
 ij_kotlin_name_count_to_use_star_import_for_members = 2147483647
 ij_kotlin_packages_to_use_import_on_demand = unset
 ktlint_code_style = intellij_idea
-ktlint_function_naming_ignore_when_annotated_with = Composable
-ktlint_standard_function-expression-body = disabled
```

Comment:
I would suggest leaving these 2 rules here, they'll be honored by Ktlint CLI too.

Reply:
We don't use the Ktlint CLI, do we?

Reply:
Yeah, but it works well for both CLI and Spotless.

Reply:
Ok, will revert changes in this file